src/components/Modal/GroupMemberListModal.tsx

```diff
@@ -20,9 +31,8 @@ const GroupMemberListModal = ({ isOpen, handleModal }: GroupMemberListProps) =>
       </DialogHeader>
       <DialogBody className=' h-96 overflow-y-auto'>
         <List>
-          {/* TODO: 키 값 여기도 처리해주세요 */}
-          {groupMember.map((member) => (
-            <ListItem key={member}>{member}</ListItem>
+          {data.nickNames.map((nickName: string) => (
+            <ListItem key={uuidv4()}>{nickName}</ListItem>
```

Comment:
`data`가 아직 api 응답 받아오지 못한 경우 `undefind` 값을 가지기 때문에 아래와 같이 `nickNames`을 받아오지 못해 에러가 발생하는 경우가 있는 것 같아요!

<img width="400" alt="image" src="https://github.com/Step3-kakao-tech-campus/Team8_FE/assets/78250089/f1fbe69e-4c92-4615-b5bc-d9e1bd7ad780">

옵셔널 체이닝으로 주면 좋을 것 같습니다!

```suggestion
          {data?.nickNames.map((nickName: string) => (
            <ListItem key={uuidv4()}>{nickName}</ListItem>
```

Reply:
아 이건 위에서 `isLoading` 한번 걸러줘서 아직 api 응답 받아오지 못한 경우에 대해서는 처리가 되고 있겠군요! isError인 상황에서는 아마 이부분 걸릴 것 같은데 안전빵으로 옵셔널 처리해주는 게 어떨까해요!